kibble/api/films.go

```diff
@@ -266,6 +267,7 @@ type filmV2 struct {
 	SeoKeywords     string                      `json:"seo_keywords"`
 	SeoDescription  string                      `json:"seo_description"`
 	CustomFields    map[string]interface{}      `json:"custom"`
+	Refs            map[string]interface{}      `json:"refs"`
```

Comment:
I think we should constrain to a new type instead of `interface` since we are specifying the keys that will be available, and they are not a random collection of client generated garbage.

```suggestion
	// Using "ref" here but whatevs the naming convention should be - refV1?
	Refs            map[string]ref{}      `json:"refs"`
```